modtools/moddable-gods.lua

```diff
@@ -62,10 +56,20 @@ if args.gender == 'male' then
  gender = 1
 elseif args.gender == 'female' then
  gender = 0
+elseif args.gender == "neuter" then
+ gender = -1
```

Comment:
I realize this is unrelated, but if these are caste raw indices, you might want to check the caste raws to make sure you're getting the gender right. I'm not sure if there's an easy way to do that, though.

Reply:
Nope, these are directly setting sex--caste is always set to "0", which is *usually* the FEMALE caste on most creatures, but I don't think it'll cause any odd behavior

Reply:
Gotcha. We really should make that a proper enum at some point (or if it is already, a lot of occurrences don't use it).